src/api/app/controllers/trigger_workflow_controller.rb

```diff
@@ -3,7 +3,7 @@ class TriggerWorkflowController < TriggerController
   include ScmWebhookPayloadDataExtractor
 
   # We don't need to validate that the body of the request is XML. We receive JSON
-  skip_before_action :validate_xml_request, :set_project_name, :set_package_name, :set_project, :set_package, :set_object_to_authorize, :set_multibuild_flavor
+  skip_before_action :validate_xml_request, :set_project_name, :set_package_name, :set_project, :set_package, :set_object_to_authorize, :set_multibuild_flavor, :check_token_class
```

Comment:
I would like to fix this whole line [here](https://trello.com/c/s3rXELVj/316-stop-using-inheritance-on-workflowtriggercontroller)